README.md

```diff
@@ -1,47 +1,75 @@
-# conu
-__CON__ -tainers __U__ -tils
-
-It provides low level __API__ for docker image/container testing.
-It is based on bash commands because primary target is to __test it
-as customers will__ use it.
-That's why it does not use python docker bindings.
-It provides python classes for:
-- __docker__ module
-  - __Image__ - image handling
-  - __Container__ - docker container handling
-- __utils__ module
-  - __Volume__ - it is mktemd utility on steroids
-  - __Probe__ - provides, simple port/file checks and waiting for this events
+# CONU
+
+Conu (CONtainer Utils) provides low level __API__ for image/container testing.
+TODO - find better punchline :)
 
 # Installation
 
-## Prerequisites
-- system packages providing binaries: `docker`, `setfacl`, `chmod`, `chcon`
-```
-$ dnf -y install acl coreutils docker
-```
-- python modules inside: No specific requirements now
+You can try out CONU in your local machine by running:
 
-## From source Code
-- Commands in git directory:
-```
-$ sudo make install
+```commandline
+git clone https://github.com/fedora-modularity/conu
+sh -c conu/requirements.sh
+pip3 install --user ./conu
 ```
 
-## COPR repo
-- Enable copr repo
-```
-dnf -y copr enable __MISSING__
-```
-- Install package
-```
-dnf -y install conu
-```
+More info [here](http://conu.readthedocs.io/en/latest/installation.html).
 
-# How to use it
+# List of features
+## Image
+- load, pull, mount and remove
+- get metadata
```

Comment:
What kind of metadata do you mean? Let's be precise. Some hint would be nice for users.